test/test_process.cpp

```diff
@@ -0,0 +1,23 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+
+#include <string>
+
+#include "rcpputils/process.hpp"
+
+TEST(TestProcess, test_get_executable_name) {
+  EXPECT_STREQ("test_process", rcpputils::get_executable_name().c_str());
```

Comment:
@cottsay fyi: I'd expect 
```suggestion
  EXPECT_EQ("test_process", rcpputils::get_executable_name());
```
to work the same.

Reply:
You're totally right, nice catch.